userdocs/src/usage/schema.js

```diff
@@ -218,6 +193,7 @@ function anchor(path, label, firstOfListType) {
             firstOfListType ? "visible" : "hidden"
         }">- </span>`;
     }
-    return html`${unsafeHTML(listPrefix)}<a class="anchor" id="${path}"></a
-        ><a class="key" href="#${path}">${label}</a>`;
+    return html`${unsafeHTML(listPrefix)}<a class="key" href="#${path}"
+            >${label}</a
+        >`;
```

Comment:
nit: Isn't this way of breaking the lines a bit weird?

Reply:
Yeah I think it was from `prettier`. The line was too long but there can't be any line breaks inside the tags. I'll check if it can be improved

Reply:
Yeah it could be a one line or with parenthesis but if it's something formatted automatically no need to fiddle with it too much, i just thought it was weird.

Reply:
done!

Reply:
nice!